src/app/pages/services/services.component.html

```diff
@@ -29,14 +29,7 @@
           *ngIf="serviceLoadingMap.get(service.service); else slider"
           matTooltipPosition="right"
           [diameter]="30"
-          [matTooltip]="
-            (service.state === ServiceStatus.Running
-              ? 'Stopping'
-              : 'Starting'
-            )
-              | uppercase
-              | translate
-          "
+          [matTooltip]="(service.state === ServiceStatus.Running ? 'Stopping' : 'Starting') | uppercase | translate"
```

Comment:
Both old and new string won't actually be correctly parsed by translate message extractor.
It should be something like '? (string | translate) : (another string | translate)'